content/docs/command-reference/repro.md

```diff
@@ -281,10 +281,11 @@ Running stage 'count' with command:
 Updating lock file 'dvc.lock'
 ```
 
-The change in the `text.txt` file is ignored as it is a dependency in the
-`filter` stage which did not get updated in the above command. This is because
-the `filter` stage happens before the `count` stage in the pipeline (shown in
-the following figure).
+The change in the `text.txt` file is ignored because that file is a dependency
+in the `filter` stage, which did not get updated in the above command. This is
+because `filter` happens before `count` in the pipeline (shown below) and the
+`--downstream` option only execute the stages after a given target stage
```

Comment:
Should the line be "... and using the --downstream option..." ?

Reply:
```suggestion
because `filter` happens before `count` in the pipeline (shown below) and with the
`--downstream` option, `dvc repro` only executes the stages after a the target stage.
```

Reply:
But I think the other alternative I suggested in https://github.com/iterative/dvc.org/pull/1624#discussion_r462086427 is better, actually:

> or update the paragraph before the code block where --downstream is mentioned, to explain what it does (very briefly) please

Reply:
> But I think the other alternative I suggested in #1624 (comment) is better, actually:

Okay. Updated accordingly.

I see that you added a full stop after "the target stage", so don't we need to mention that the target stage is also included when using `--downstream` option or is it evident from the output above this para?